lib/volume_handler.coffee

```diff
@@ -5,6 +5,10 @@ class VolumeHandler
   constructor: (initial_step = 5) ->
     @exec = require('child_process').exec
     @set initial_step
+    
+    # Adding somewhere to store a held volume
+    # and the reason it was held
+    @sticky_volume = status: "none", value: 0
```

Comment:
Maybe move this into the request handler to make it more obvious as global state?